src/graph/graph.py

```diff
@@ -5,7 +5,7 @@
 from .link import Link
 import networkx as nx
 import matplotlib.pyplot as plt
-
+import json
 
 class Graph(object):
```

Comment:
Con el método inverso comprobar si nos pasan un path a un json, para conformar un obj.

Meter un if y si json_path is none, aplicar la lógica ya existente en el constructor de la clase. En caso contrario, si nos pasan un json, damos prioridad a conformar el obj desde el json